hist/hist/src/TGraphDelaunay2D.cxx

```diff
@@ -17,13 +17,17 @@ ClassImp(TGraphDelaunay2D);
 
 /** \class TGraphDelaunay2D
     \ingroup Graphs
-TGraphDelaunay2D generates a Delaunay triangulation of a TGraph2D. This
-triangulation code derives from an implementation done by Luke Jones
-(Royal Holloway, University of London) in April 2002 in the PAW context.
-
-This software cannot be guaranteed to work under all circumstances. They
-were originally written to work with a few hundred points in an XY space
-with similar X and Y ranges.
+TGraphDelaunay2D generates a Delaunay triangulation of a TGraph2D.
+The algorithm used for finding the triangles is based on on
+**Triangle**, a two-dimensional quality mesh generator and
+Delaunay triangulator from Jonathan Richard Shewchuk.
+See [http://www.cs.cmu.edu/~quake/triangle.html]
+The ROOT::Math::Delaunay2D class provides a wrapper for using
+the **Trinagle** library.
+
+This implementation provides large improvements in terms of computational performances
+compared to the legacy one available in TGraphDelaunay, and it is by default
+used in TGraph2D.
```

Comment:
Should we tell the legacy one can be accessed via the "old" option ?

Reply:
Yes, good point! I see the option "old" is available for TGraph2D::GetHistogram and also for TGraph2D::Paint(), this means that is then also available for TGraph2D::Draw, correct ?

Reply:
yes